src/aura/CMP_Acc_Naming_Dropdown/CMP_Acc_Naming_DropdownHelper.js

```diff
@@ -0,0 +1,94 @@
+({
+    init : function(component) {
+        //Retrieving admin account naming options
+        this.getAdminAccNameFormatOptions(component);
+        this.getHHAccNameFormatOptions(component);
+    },
+    onSelectChange : function(component) {
+        var selectedVal = component.find("nameFormatDropDown").get("v.value");
+        var prefix = component.get("v.namespacePrefix");
+        if (prefix === 'hed__')
```

Comment:
There are  a lot of hard code namespace "hed__" in the code. It is a standard way to do it in lightning component?

Reply:
Thanks Wei for bringing this up. I think this is how we use custom label in HEDA to deal with managed/unmanaged code. I agree it is now a best practice but seems the only solution I can think of. @ceiroa what do you think on this?

Reply:
Yes, we need to check the prefix, though this code could be cleaned up a little. 

1. You could use `https://github.com/SalesforceFoundation/HEDAP/blob/master/src/classes/UTIL_Namespace.cls` to get the namespace. Or, EVEN BETTER, since STG_CMP_System.cmp extends STG_CMP_Base, the namespace should already be available, as it's a property in that base class: https://github.com/SalesforceFoundation/HEDAP/blob/master/src/aura/STG_CMP_Base/STG_CMP_Base.cmp#L3.

2. For consistency, since the opening and closing curly braces are in the same line as the `else`, you could put them in the same line as the `if`. Also, you could make it more compact:

```
if (prefix === 'hed__') {
    var accNamingOther = $A.get("$Label.hed.acctNamingOther");
} else {
    var accNamingOther = $A.get("$Label.c.acctNamingOther");
}

if(selectedVal === accNamingOther) {
    component.set("v.otherDisplay", true);
} else {
    component.set("v.otherDisplay", false);
}
```
That way you avoid repeating the if-else.